lib/WasmReader/SExprParser.h

```diff
@@ -31,7 +31,11 @@ namespace Wasm
     public:
         SExprParser(PageAllocator * alloc, LPCUTF8 source, size_t length);
 
-        virtual WasmOp ReadFromModule() override;
+        virtual void InitializeReader() override;
+        virtual bool ReadNextSection(SectionCode nextSection) override;
+        virtual bool ProcessSection(SectionCode nextSection) override;
+        // todo:: Remove and use ReadNextSection and ProcessSection instead
+        virtual WasmOp ReadFromModule();
```

Comment:
Do we want to keep the override specifier? But since ReadFromModule() is on its way out, it is probably a moot point.